### PR TITLE
docs: Add notes on installing custom components

### PR DESCRIPTION
**Pull request by mrtmm**
Add notes and examples to the README for installing custom
components and brand packages to the MFEs.

### CODE REVIEW

**Review by BbrSofiane — APPROVED** [review]
Looks good to me. (Thanks for your patience I should have reviewed this a while back!)

File: README.rst [hunk 1]
```diff
@@ -141,6 +141,35 @@ To change the MFEs logos from the default to your own logos, override the corres
         LOGO_WHITE_URL=<URL>/logo-white.svg
         FAVICON_URL=<URL>/favicon.ico
 
+To install custom components for the MFEs, such as the `header <https://github.com/openedx/frontend-component-header>`_ and `footer <https://github.com/openedx/frontend-component-footer>`_, override the components by adding a patch to ``mfe-dockerfile-post-npm-install`` in your plugin:
+::
+
+    patches:
+        ...
+        mfe-dockerfile-post-npm-install |
```
Comment (regisb) [inline README.rst:149]: Isn't a ":" missing here?

```
mfe-dockerfile-post-npm-install: |
    ...
```

Same comment for the patches below.

File: README.rst [hunk 2]
```diff
@@ -141,6 +141,35 @@ To change the MFEs logos from the default to your own logos, override the corres
...
+
+The same applies to installing a custom `brand <https://github.com/openedx/brand-openedx>`_ package:
+::
+
+    patches:
+        ...
+        mfe-dockerfile-post-npm-install |
+            RUN npm install '@edx/brand@git+https://github.com/edx/brand-edx.org.git'
+
+In case you need to install components from a private NPM registry, you can append the ``--registry`` option to your install statement or add a ``npm config set`` command to the plugin.
```
Comment (regisb) [inline README.rst:163]: Would you mind adding a different title to this section, such as "Installing from a private npm registry", or something similar?

**Review by regisb — CHANGES_REQUESTED** [review]
These are great changes, thanks a lot for your contribution. Can you just please have a look at my comments?

File: README.rst (hunk 1 above)
Comment (mrtmm) [reply README.rst:149]: Ah, yes, I'll fix that, thanks!

File: README.rst (hunk 2 above)
Comment (mrtmm) [reply README.rst:163]: Sure, will do.

**Pull request merged**